src/components/ErrorList.jsx

```diff
@@ -16,7 +16,7 @@ function ErrorList({errors, onErrorClick, language}) {
       key={[error.reason, error.row]}
       onClick={partial(
         onErrorClick,
-        language,
+        `editor.${language}`,
```

Comment:
I would actually keep this the way it was, since the `ErrorList` component shouldn’t need to know the details of the store—from its standpoint, it’s just giving some information about an error being clicked. We can decorate it in the `Workspace#_handleErrorClick` method, which has direct knowledge of the `focusLine` action and what format it expects arguments in.

Reply:
Makes sense. I also just noticed the `Workspace#_handleErrorClick` method looks like it is never used. The `onErrorClick` in the `ErrorReport` Container seems to take care of this.

Reply:
Nice—feel free to remove the unused method as well!